src/main/java/org/prebid/server/bidder/README.md

```diff
@@ -19,4 +19,5 @@ for review and approval of the ported adapter:
 - lifestreet
 - pubmatic
 - pulsepoint
-- sovrn
\ No newline at end of file
+- sovrn
+- openx
```

Comment:
It seems that we support alphabet order here, so openx should be after lifestreet

Reply:
indeed, tnx